packages/wonder-blocks-i18n/src/components/i18n-inline-markup.stories.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import * as React from "react";
+
+import {I18nInlineMarkup} from "./i18n-inline-markup.js";
+
+export default {
+    title: "Translations/I18nInlineMarkup",
+    parameters: {
```

Comment:
You might want to include `component: I18nInlineMarkup` here in case you want to expose the Props table.

Reply:
Oh one more thing 🙏 ... could you please move this file inside the `__docs__` folder?